packages/devextreme/testing/tests/DevExpress.ui.widgets/splitter.tests.js

```diff
@@ -882,6 +882,15 @@ QUnit.module('Pane sizing', moduleConfig, () => {
                 { targetButton: 'prev', resizeHandleIndex: 0, expectedLayout: ['10.1626', '44.9187', '44.9187'] },
                 { targetButton: 'next', resizeHandleIndex: 0, expectedLayout: ['27.5407', '27.5407', '44.9187'] },
             ]
+        },
+        {
+            items: [ { collapsible: true, collapsedSize: 50, size: 100, minSize: 100, maxSize: 100 }, { collapsible: true, size: 100 }],
```

Comment:
I would prefer to add extra configuration with three panes, like the sample in the original issue, to be sure that distribution between panes works correctly